websites/Y/Yandex.Translate/dist/metadata.json

```diff
@@ -0,0 +1,41 @@
+{
+    "author": {
+      "name": "Striker",
+      "id": "215509157837537280"
+    },
+    "url": [
+      "translate.yandex.com",
+      "translate.yandex.ru",
+      "ceviri.yandex.com.tr",
+      "translate.yandex.ua"
+    ],
+    "description": {
+      "en": "Yandex's free service instantly translates words, phrases, and web pages between English and 98 other languages!"
+    },
+    "service": "Yandex.Translate",
```

Comment:
Why with dot? Just use space? Same in folder name

Reply:
in the documentation for the website/the support page it is referenced to as Yandex.Translate, same with outside of the website (attached screenshots) 
![Screenshot_20210115-170937_Chrome Beta](https://user-images.githubusercontent.com/34070194/104757134-95618e00-5754-11eb-8ba3-41b94b7e099f.jpg)
![Screenshot_20210115-171010_Chrome Beta](https://user-images.githubusercontent.com/34070194/104757148-98f51500-5754-11eb-9642-347833b61e7f.jpg)

Reply:
I was also keeping to the name of the original request (#2699)

Reply:
bruh

Reply:
yeah i know, consistency definitely isn't their thing it seems as Yandex Music has no dot. weird.